pinakes/main/catalog/serializers.py

```diff
@@ -95,18 +94,17 @@ class CopyPortfolioSerializer(serializers.Serializer):
     )
 
 
-class PortfolioItemSerializer(serializers.ModelSerializer):
+class PortfolioItemSerializerBase(serializers.ModelSerializer):
     """PortfolioItem which maps to a Controller Job Template
     via the service_offering_ref"""
 
     icon_url = serializers.SerializerMethodField(
         "get_icon_url", allow_null=True
     )
 
-    metadata = serializers.JSONField(
-        read_only=True,
-        help_text="JSON Metadata about the portfolio item",
-        default={},
+    metadata = MetadataSerializer(
+        skip_user_capabilities=True,
```

Comment:
@cutwater Is this temporary, we would need user_capabilities for PortfolioItems.

Reply:
This is a base serializer class, it does not include user capabilities intentionally, as it is used as a nested serializer. 
The metadata field is overridden in `PortfolioItemSerializer`, which includes user capabilities and used in `PortfolioItemViewSet`.

Reply:
I tested locally and saw the user capabilities get returned from the portfolio_items API call, so I can confirm that they are there!